src/vendor/wordpress/core-blocks.stories.mdx

```diff
@@ -397,12 +389,28 @@ closing wrapper tag.
     name="Image"
     argTypes={{
       alignment: {
-        options: alignmentClasses,
+        options: {
+          None: '',
+          Left: 'alignleft',
+          Center: 'aligncenter',
+          Right: 'alignright',
+          Full: 'alignfull',
+          Wide: 'alignwide',
+        },
+        defaultValue: '',
+        control: { type: 'select' },
+      },
+      style: {
+        options: {
+          None: 'is-style-default',
+          Outlined: 'is-style-outlined',
+        },
+        defaultValue: 'is-style-default',
         control: { type: 'select' },
       },
     }}
   >
-    {(args) => blockImageDemo({ class: args.alignment })}
+    {(args) => blockImageDemo({ alignment: args.alignment, style: args.style })}
```

Comment:
I changed `class` to `alignment` because we're setting multiple classes now, and it made the logic in the twig file easier to understand.